collect_app/src/main/java/org/odk/collect/android/logic/FormDetails.java

```diff
@@ -17,37 +17,60 @@
 import java.io.Serializable;
 
 public class FormDetails implements Serializable {
-    /**
-     *
-     */
     private static final long serialVersionUID = 1L;
 
-    public final String errorStr;
-
-    public final String formName;
-    public final String downloadUrl;
-    public final String manifestUrl;
-    public final String formID;
-    public final String formVersion;
-
+    private String errorStr;
+    private String formName;
+    private String downloadUrl;
+    private String manifestUrl;
+    private String formID;
+    private String formVersion;
+    private boolean isNewerFormVersionAvailable;
```

Comment:
It feels odd to me to have these two booleans as fields since they don't represent permanent state of the form. I don't have a great alternative yet but wanted to mention it.

Reply:
I suppose these `FormDetail` objects only exist for the download list, right?

Reply:
Yes, it's used only for downloading. it might look a little bit strange but it's the easiest way I don't need to create a separate collection with those values.